modules/super-admin/components/product-listing/ProductListingNavbar.tsx

```diff
@@ -1,15 +1,45 @@
 import { ArrowUp } from 'iconsax-react';
 import Link from 'next/link';
+import axios from 'axios';
+import { useState, useEffect } from 'react';
+import { ProductStatistics } from '../../../../@types';
 
 const ProductsListingNavbar = () => {
+  const [productStats, setProductStats] = useState<ProductStatistics | null>(null);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const authToken =
+          'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6ImE3ODlmMzVlLTAwNjQtNDM1Yy1hMTI5LTI3OGY5YzFhYTFkYiIsImlhdCI6MTY5NzE0MjQwMX0.Np4b4179X0XfbmBlN9ev_EYc8DLPyxtW7UOcf3b4dxI';
```

Comment:
please remove this hardcode token and make it dynamic, coz the auth team has connected the login with the admin dashboard meaning you can get the token n set it to local storage after login. So make the token dynamic by getting it from the localStorage.